src/pe-impersonate-utility/README.md

```diff
@@ -0,0 +1,65 @@
+# Impersonate Utility
+
+## Description
+
+With this widget you will be able to impersonate Users from a Service Portal page.
+
+## Screenshot
+
+![Impersonate Utility](https://raw.githubusercontent.com/platform-experience/serviceportal-widget-library/master/src/pe-impersonate-utility/images/pe-impersonate-utility.png)
+
+## Additional Information/Notes
+
+> None
+
+## Installation
+
+Download and install update set **[pe-impersonate-utility.u-update-set.xml](https://github.com/platform-experience/serviceportal-widget-library/blob/master/src/pe-impersonate-utility/pe-impersonate-utility.u-update-set.xml)**
+
+After installation, the widget can be accessed via the `Service Portal > Widgets` section for use and customization.
+
+* SN Product Documentation - ['Load a customization from a single XML file'](https://docs.servicenow.com/bundle/kingston-application-development/page/build/system-update-sets/task/t_SaveAnUpdateSetAsAnXMLFile.html)
+
+## Configuration
+
+> Follow these steps to configure Impersonation Users.
+
+* In left nav search for **Impersonation**. You can all the parts that make the Widget Packs app.
```

Comment:
Please address the missing words in this instruction. Once added, it will make things easier to follow.